recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -2,8 +2,17 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 FILESEXTRAPATHS_prepend := "${THISDIR}/../../inc:"
 
 do_fetch[depends] += "domd-agl-demo-platform:do_${BB_DEFAULT_TASK}"
-do_fetch[depends] += "domu-image-android:do_${BB_DEFAULT_TASK}"
-do_fetch[depends] += "domu-image-fusion:do_${BB_DEFAULT_TASK}"
+
+XT_GUESTS_BUILD ?= "doma domf"
+XT_GUESTS_INSTALL ?= "doma domf"
+
+python __anonymous () {
+    guests = d.getVar('XT_GUESTS_BUILD', True).split()
+    if "doma" in guests :
```

Comment:
I would better have not an "add domain option", but rather "remove domain from the build", e.g. you have a default set of domains to be built and by remove option you can remove those you don't want.
So, the default product builds as usually

Reply:
A line above this function is defined a default set of domains. In my case I want to set `XT_GUESTS_BUILD = "doma domr"`, in such a case 'removing from default' semantic would look weird.

Reply:
ok, my bad